runtime/moonriver/src/lib.rs

```diff
@@ -1229,6 +1232,8 @@ construct_runtime! {
 		ProxyGenesisCompanion: pallet_proxy_genesis_companion::{Pallet, Config<T>} = 35,
 
 		// Sudo was previously index 40
+		OpenTechCommitteeCollective:
```

Comment:
Why 46 and not 73 like other council stuff ?

Reply:
crap missed it, was trying to keep the indices same as moonbase.